doc/source/serve/ml-models.rst

```diff
@@ -44,6 +44,10 @@ dive.
 Model Composition
 =================
 
+.. note::
+  Serve recently added experimental first-class model composition API. Please
+  take a look at :ref:`Pipeline API <serve-pipeline-api>` and try it out!
```

Comment:
```suggestion
  take a look at the :ref:`Pipeline API <serve-pipeline-api>` and try it out!
```

Reply:
Please add a call to action for people to file issues or post on forum w/ questions.